### PR TITLE
Feature/registry listener

### PR DESCRIPTION
**Pull request by williamfeng323**
**What this PR does**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/adapter/dubboregistry/registry/zookeeper/application_listener.go [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+					z.servicesPath, MaxFailTimes)
+				return
+			}
+			delayTimer.Reset(ConnDelay * time.Duration(failTimes))
+			<-delayTimer.C
+			continue
+		}
+		failTimes = 0
+		tickerTTL := defaultTTL
+		ticker := time.NewTicker(tickerTTL)
```
Comment (AlexStocks) [inline pkg/adapter/dubboregistry/registry/zookeeper/application_listener.go:114]: defer ticker.Stop()

File: pkg/adapter/dubboregistry/registry/zookeeper/application_listener_test.go [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package zookeeper
+
+//
```
Comment (AlexStocks) [inline pkg/adapter/dubboregistry/registry/zookeeper/application_listener_test.go:20]: 这个文件全部注释掉，为啥？

File: pkg/adapter/dubboregistry/registry/zookeeper/interface_listener.go [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+					z.path, MaxFailTimes)
+				return
+			}
+			delayTimer.Reset(ConnDelay * time.Duration(failTimes))
+			<-delayTimer.C
+			continue
+		}
+		failTimes = 0
+		tickerTTL := defaultTTL
+		ticker := time.NewTicker(tickerTTL)
```
Comment (AlexStocks) [inline pkg/adapter/dubboregistry/registry/zookeeper/interface_listener.go:105]: defer ticker.Stop()

File: pkg/adapter/dubboregistry/registry/zookeeper/application_listener_test.go (hunk 2 above)
Comment (ztelur) [reply pkg/adapter/dubboregistry/registry/zookeeper/application_listener_test.go]: 应用级服务发现的相关逻辑暂时去掉了，放在下一个迭代

**Review by ztelur — APPROVED** [review]

**Review by xiaoliu10 — APPROVED** [review]

**Review by AlexStocks — APPROVED** [review]

**Pull request merged**